birdhouse/optional-components/generic_bird/postgres/docker-entrypoint-initdb.d/create-wps-databases.sql.template

```diff
@@ -0,0 +1 @@
+CREATE DATABASE ${BIRDHOUSE_GENERIC_BIRD_POSTGRES_DB};
```

Comment:
missing `.gitignore` update matching the generated file of this new template file?